admin-manual.md

```diff
@@ -0,0 +1,187 @@
+# Running a MISO Instance
+MISO requires some configuration directly in the source code. While we plan to
+change this over time, running an instance of MISO will require building and
+deploying a fork of the code base with the customisations.
+
+# Creating a Fork and Setting Configuration
+Use the GitHub interface or a private instance to create a forked repository.
+
+Proceed to set up a build environment, then edit
+`miso-web/src/main/webapp/WEB-INF/miso-config.xml` and set the naming schemes.
+The naming schemes will determine how MISO checks if object names (sample,
+library, etc) are valid. If you do not want to use one of the supplied ones
+(TGAC's standard, OICR's standard, or no checks), it is likely you will have to
+write one specific to your organisation.
+
+# Setting Up the Build Environment
+One or machines should be set up to build MISO. This should probably not be on
+the production server. A typical Linux, Windows, or MacOS system will work.
+
+You will need:
+
+- [Maven 3.0.5](http://maven.apache.org/download.html) or later
+- Git
+
+For development purposes, Eclipse is recommended and it might be useful to set
+up the server environments for testing. There is an automatic code formatting
+configuration available for Eclipse.
+
+Locally, create a checkout:
+
+   git clone you@server:your-miso.git
+   git remote add tgac git@github.com:TGAC/miso-lims.git
+
+# Setting Up the Database Server
+The database server needs to have [mySQL 5](https://www.mysql.com/). The tool
+[Flyway](https://flywaydb.org/) must also be present to migrate the databse as
+the application is developed, but it can be installed on a different server so
+long as it can access the database server.
+
+Defaults passwords should be changed.
```

Comment:
Okay. I had meant the database password. I've fixed that and commented about changing the admin and notification passwords in the security section.